flow/data_pipeline/datapipeline_test.py

```diff
@@ -0,0 +1,37 @@
+"""functions that calculates the expected result for testing."""
```

Comment:
Can you move this to the tests folder?

Reply:
Oh unless, it specifically requires some sort of AWS authentication, in which case, might be best to leave it.

Reply:
@AboudyKreidieh thoughts? I feel like most of our tests should be runnable by anyone.

Reply:
i agree this should probably be a publicly available test (if possible). one wrinkle is when we have Toyota NDA restricted models though -- those will not be able to have tests, I guess.

Reply:
i'd avoiding writing tests for anything that can expose NDA things, but for everything else feel free to move to the tests folder